-------------------------------------------------------------------------------
Test set: Programmer.testing.MembuatTest.CalculatorTest
-------------------------------------------------------------------------------
Tests run: 7, Failures: 3, Errors: 0, Skipped: 0, Time elapsed: 0.023 s <<< FAILURE! - in Programmer.testing.MembuatTest.CalculatorTest
testDivideByZero  Time elapsed: 0.001 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: Unexpected exception type thrown, expected: <java.lang.IllegalArgumentException> but was: <java.lang.ArithmeticException>
	at Programmer.testing.MembuatTest.CalculatorTest.testDivideByZero(CalculatorTest.java:39)
Caused by: java.lang.ArithmeticException: Pembagian dengan nol tidak diperbolehkan
	at Programmer.testing.MembuatTest.CalculatorTest.lambda$testDivideByZero$0(CalculatorTest.java:39)
	at Programmer.testing.MembuatTest.CalculatorTest.testDivideByZero(CalculatorTest.java:39)

testUnderflowInteger  Time elapsed: 0 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: Expected java.lang.ArithmeticException to be thrown, but nothing was thrown.
	at Programmer.testing.MembuatTest.CalculatorTest.testUnderflowInteger(CalculatorTest.java:51)

testOverflowInteger  Time elapsed: 0 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: Expected java.lang.ArithmeticException to be thrown, but nothing was thrown.
	at Programmer.testing.MembuatTest.CalculatorTest.testOverflowInteger(CalculatorTest.java:45)

